### PR TITLE
Fix `points2binaryimage` tool

### PR DESCRIPTION
**Pull request by kostrykin**
**For the contributor:**
* [x] I have read the [CONTRIBUTING.md](https://github.com/BMCV/galaxy-image-analysis/blob/master/CONTRIBUTING.md) document.
* [x] License permits unrestricted use (educational + commercial).
* [x] This PR adds or updates a tool or tool collection.
  * [ ] This PR adds a new tool or tool collection.
  * [x] This PR updates an existing tool or tool collection.
  * [x] Tools added/updated by this PR comply with the [Naming and Annotation Conventions for Tools in the Image Community in Galaxy](https://github.com/elixir-europe/biohackathon-projects-2023/blob/main/16/paper/paper.md#conventions) (or explain why they do not).
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Review by bgruening — COMMENTED** [review]
feel free to add creators

**Comment by kostrykin** [comment]
Nice to finally have the ability to specify creators in tool wrappers!
No idea how long this have been around, but I missed it until now :)

To reduce boilerplate, I think it would be nice to have reusable creators. I tried to achieve this with macros in 8453a6f133c3d32b0845f69198f777c880dc9186.

File: macros/creators/bmcv.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<macros>
+    <xml name="creator">
+        <organization name="Biomedical Computer Vision Group" alternateName="BMCV" url="http://www.bioquant.uni-heidelberg.de/research/groups/biomedical_computer_vision.html" />
```
Comment (bgruening) [inline macros/creators/bmcv.xml:3]: probably good to have a yield here.

Comment (kostrykin) [reply macros/creators/bmcv.xml:3]: Hm I have looked into [`yield`](https://planemo.readthedocs.io/en/latest/writing_advanced.html#parameterizing-xml-macros-with-yield) but I do no see in what way you mean to use it here?

**Pull request merged**